x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -82,25 +82,27 @@ export function getServiceColumns({
       ? [
           {
             field: ServiceInventoryFieldName.AlertsCount,
-            name: '',
-            width: `${unit * 5}px`,
+            name: 'Active alerts',
```

Comment:
We should add i18n here
```suggestion
            name: i18n.translate('xpack.apm.servicesTable.alertsColumnLabel', {
        defaultMessage: 'Active alerts',
      }),
```

Reply:
After adding that you'll need to run this script to update the i18n files `node scripts/i18n_check --fix`